Makefile

```
@@ -11,12 +11,18 @@ VWC_ANNOTATION ?= managed.openshift.io/inject-cabundle-from
 
 IMG ?= quay.io/cshereme/python3-webhookbase
 
+CONTAINER_ENGINE=$(shell which podman 2>/dev/null || which docker 2>/dev/null)
```

```suggestion
CONTAINER_ENGINE ?= $(shell which podman 2>/dev/null || which docker 2>/dev/null)
```

Can let someone override it at the shell if they want.